infinispan-remote/redis-client/src/main/java/org/infinispan/tutorial/simple/redis/RedisClientCache.java

```diff
@@ -0,0 +1,24 @@
+package org.infinispan.tutorial.simple.redis;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.api.sync.RedisCommands;
+import org.infinispan.client.hotrod.impl.ConfigurationProperties;
```

Comment:
I would avoid referencing a Hot Rod class (even if it's just a constant)

Reply:
Maybe add a "SINGLE_PORT" class variable to `Infinispan` instead or something?